### DISCUSSION TITLE
Solution for DB2 connection

### DISCUSSION THREAD

**Discussion by ShreeramM** [cypress-io/cypress#20862] [General]
Need a solution on how to connect to DB2 Database using cypress

**Comment by jjhelguero** [comment]
NPM is your friend =)
Have you looked into this [ibm-db](https://www.npmjs.com/package/ibm_db)?

**Reply by ShreeramM** [reply]
yes i tried looking into that but the DB Connection i am not able to find it using cypress for DB2

**Reply by jjhelguero** [reply]
You'll have to create a [plugin](https://docs.cypress.io/api/plugins/writing-a-plugin#Plugins-API) with ibm-db and then call it using `cy.task()`.
